Project.toml

```diff
@@ -25,11 +25,11 @@ Thermodynamics = "b60c26fb-14c3-4610-9d3e-2d17fe7ff00c"
 
 [compat]
 CLIMAParameters = "0.4, 0.5, 0.6, 0.7"
-ClimaAtmos = "0.17.1"
+ClimaAtmos = "0.18"
```

Comment:
Maybe you can move to 0.19 directly?

I don't think there's going to be any challenge.